rcl_yaml_param_parser/Doxyfile

```diff
@@ -0,0 +1,29 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rcl_yaml_param_parser"
+PROJECT_NUMBER         = master
+PROJECT_BRIEF          = "Package containing various utility types and functions for C."
```

Comment:
This seems like a copy of rcutils's package description.

Reply:
I copied the description from the package.xml. I will update the description in a separated PR.

Reply:
https://github.com/ros2/rcl/pull/637